crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -0,0 +1,381 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use crate::{cli, cli::traits::Cli as _, cli::Cli, style::style};
+use clap::{Args, ValueEnum};
+use cliclack::{confirm, input};
+use pop_common::Profile;
+use pop_parachains::{
+	binary_path, build_parachain, export_wasm_file, generate_genesis_state_file,
+	generate_plain_chain_spec, generate_raw_chain_spec, is_supported, replace_chain_type,
+	replace_protocol_id, replace_relay_spec,
+};
+use std::{env::current_dir, fs::create_dir_all, path::PathBuf};
+#[cfg(not(test))]
+use std::{thread::sleep, time::Duration};
+use strum::{EnumMessage, VariantArray};
+use strum_macros::{AsRefStr, Display, EnumString};
+
+const DEFAULT_SPEC_NAME: &str = "template-chain-spec.json";
+
+#[derive(
+	AsRefStr,
+	Clone,
+	Default,
+	Debug,
+	Display,
+	EnumString,
+	EnumMessage,
+	ValueEnum,
+	VariantArray,
+	Eq,
+	PartialEq,
+)]
+/// Supported chain types for the resulting chain spec.
+pub(crate) enum ChainType {
+	// A development chain that runs mainly on one node.
+	#[default]
+	#[strum(
+		serialize = "Development",
+		message = "Development",
+		detailed_message = "Meant for a development chain that runs mainly on one node."
+	)]
+	Development,
+	// A local chain that runs locally on multiple nodes for testing purposes.
+	#[strum(
+		serialize = "Local",
+		message = "Local",
+		detailed_message = "Meant for a local chain that runs locally on multiple nodes for testing purposes."
+	)]
+	Local,
+	// A live chain.
+	#[strum(serialize = "Live", message = "Live", detailed_message = "Meant for a live chain.")]
+	Live,
+}
+
+#[derive(
+	AsRefStr,
+	Clone,
+	Default,
+	Debug,
+	Display,
+	EnumString,
+	EnumMessage,
+	ValueEnum,
+	VariantArray,
+	Eq,
+	PartialEq,
+)]
+/// Supported relay chains that can be included in the resulting chain spec.
+pub(crate) enum RelayChain {
+	#[strum(
+		serialize = "kusama",
+		message = "Kusama",
+		detailed_message = "Polkadot's canary network."
+	)]
+	Kusama,
+	#[strum(
+		serialize = "kusama-local",
+		message = "Kusama Local",
+		detailed_message = "Local configuration for Kusama network."
+	)]
+	KusamaLocal,
+	#[strum(serialize = "rococo", message = "Rococo", detailed_message = "Parity's test network.")]
+	Rococo,
```

Comment:
I'm not sure we should include Rococo/Local with its imminent retirement - perhaps replace with Westend/Local so this list is more accurate? Description should maybe read 'Parity's test network for protocol testing' or something.

Reply:
Makes sense, though keep in mind that mostly everybody testing coretime is doing so in some flavor of Rococo. Although Paseo should integrate coretime rather soon, so I think we can get rid of Rococo anyway.

Reply:
Good point. Maybe we decorate them as deprecated? Both via https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute and via the description which appears in the UI.

Reply:
It feels better not including it than including them new with a deprecated flag.